tests/integration/docusaurus/connecting_to_your_data/cloud/gcs/pandas/configured_yaml_example.py

```diff
@@ -18,6 +18,8 @@
         class_name: ConfiguredAssetGCSDataConnector
         bucket_or_name: <YOUR_GCS_BUCKET_HERE>
         prefix: <BUCKET_PATH_TO_DATA>
+        gcs_options:
+            delimiter: /
```

Comment:
How about just passing the delimiter directly via the delimiter param?

```suggestion
        delimiter: /
```